spec/lucky/input_helpers_spec.cr

```diff
@@ -70,6 +70,16 @@ describe Lucky::InputHelpers do
     HTML
   end
 
+  it "renders file inputs" do
+    view.file_input(form.first_name).to_s.should contain <<-HTML
+    <input type="file" id="user_first_name" name="user:first_name" value="My name"/>
+    HTML
+
+    view.file_input(form.first_name, class: "cool").to_s.should contain <<-HTML
```

Comment:
One there is support in `LuckyRecord` this helper should probably restrict the field to `LuckyRecord::Field(Tempfile)` or something. But for now I think this is good 👍

Reply:
Noted